tests/dialects/test_starrocks.py

```diff
@@ -91,7 +90,7 @@ def test_unnest(self):
                 "spark": r"""SELECT id, t.type, t.scores FROM example_table LATERAL VIEW INLINE(ARRAYS_ZIP(SPLIT(type, CONCAT('\\Q', ';', '\\E')), scores)) t AS type, scores""",
                 "databricks": r"""SELECT id, t.type, t.scores FROM example_table LATERAL VIEW INLINE(ARRAYS_ZIP(SPLIT(type, CONCAT('\\Q', ';', '\\E')), scores)) t AS type, scores""",
                 "starrocks": r"""SELECT id, t.type, t.scores FROM example_table, UNNEST(SPLIT(type, ';'), scores) AS t(type, scores)""",
-                "hive": UnsupportedError,
+                "hive": r"""SELECT id, t.type, t.scores FROM example_table LATERAL VIEW INLINE(ARRAYS_ZIP(SPLIT(type, CONCAT('\\Q', ';', '\\E')), scores)) t AS type, scores""",
```

Comment:
Why did these hive tests change? Did you test these queries?

Reply:
Yes. hive test has changed because `hive`  do support array of struct using inline.

ref doc: [LanguageManualUDF-inline- arrayofstructs](https://cwiki.apache.org/confluence/display/hive/languagemanual+udf#LanguageManualUDF-inline(arrayofstructs))

I'll check for some more scenarios regarding this, thanks for pointing it out.

Reply:
The issue is not `INLINE` in this case, though. It's `ARRAYS_ZIP`; IIRC, it doesn't support it.

Reply:
Okay, Let me run a few more testcases around `hive`.
If this has the issue you mentioned above I'll re-adjust the code. Thank you